src/app/app.component.ts

```diff
@@ -13,7 +14,18 @@ export class AppComponent {
   title = 'Angular Mentoring Program 2019';
   displayOverlay$: Observable<boolean>;
 
-  constructor(private readonly overlayService: OverlayService) {
+  constructor(
+    readonly overlayService: OverlayService,
+    readonly translate: TranslateService,
+    @Inject('SUPPORTED_LANGUAGES') private readonly supportedLanguages: string[],
+  ) {
     this.displayOverlay$ = overlayService.showOverlay$.asObservable().pipe(distinctUntilChanged());
+
+    const defaultLanguage = supportedLanguages[0];
```

Comment:
Since ES2015 introduced object/array destructuring, we may use `const [defaultLanguage] = supportedLanguages;` to get values from the array.

Reply:
Awesome! I'll use this.